assets/styles/style.css

```diff
@@ -0,0 +1,192 @@
+/*
+Links on CSS/SASS you should take a look at:
+
+BEM (Block, Element, Modifier)

```

Comment:
Take a look at BEM, this is a way of writing CSS following a predictable pattern for naming classes, it really helps organize your stylesheets.